yellowbrick/target/binning.py

```diff
@@ -118,15 +118,6 @@ def fit(self, y, **kwargs):
         self.draw(y)
         return self
 
-
-    def poof(self, **kwargs):
```

Comment:
Removed this method because it shouldn't be in visualizer subclasses and because it was making the image not be rendered.

Reply:
@bbengfort I found that in my audit...thanks for fixing it.